provider/lib/cronAlarm.js

```diff
@@ -85,4 +85,28 @@ module.exports = function(logger, newTrigger) {
         }
     };
 
+    // Convert string to integer in [0, 60)
+    function hashName(name) {
+        var hash = 0;
+
+        for (var i = 0; i < name.length; i++) {
+            var char = name.charCodeAt(i);
+            hash = ((hash << 5) - hash) + char;
+        }
+        hash %= 60;
```

Comment:
Can we make this also configurable at the deployment time?
So operators can choose the maximum delay.